### PR TITLE
Issue 100 industry standards

### PR DESCRIPTION
**Pull request by TheAyeshaSiddiqa**
Note: Industry Standards version handling is not included in this PR.
I'll open another PR for that after some discussion.

### CODE REVIEW

**Review by BarisSari — CHANGES_REQUESTED** [review]
Thanks for the PR, @TheAyeshaSiddiqa. I asked for minor changes. After that, I'm going to approve it.

File: app/domain/entities/industry_standard.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class IndustryStandardV2(BaseModel):
+    standard_id: str
+    standard_title: str
+    standard_version: str
+    standard_status: str
+    status_attained_date: datetime
+    review_date: datetime
+    competencies_list: list
+    description: str
```
[the remark this answers is not in the document]
Comment (TheAyeshaSiddiqa) [reply app/domain/entities/industry_standard.py:24]: yeah good idea!

**Comment by BarisSari** [comment]
It seems like there are many missing lines, @TheAyeshaSiddiqa. Can you implement API tests for new endpoints?

**Comment by TheAyeshaSiddiqa** [comment]
> It seems like there are many missing lines, @TheAyeshaSiddiqa. Can you implement API tests for new endpoints?

Sure, I can.

**Review by monkeypants — APPROVED** [review]
codecov has a few niggles but otherwise LGTM

**Comment by monkeypants** [comment]
I'll force this in (with coverage drop) and leave the branch open for more tests.

**Pull request merged**